dts/bindings/dac/st,stm32-dac.yaml

```diff
@@ -14,6 +14,15 @@ properties:
     clocks:
       required: true
 
+    pinctrl-0:
+      type: phandles
+      required: false
+      description: |
+        GPIO pin configuration for DAC output. We expect that the phandles
+        will reference pinctrl nodes, e.g.
+
+           pinctrl-0 = <&dac_out1_pa4 &dac_out1_pa5>;
+
```

Comment:
Should be dac_out2_pa5 and not dac_out1_pa5 
```
pinctrl-0 = <&dac_out1_pa4 &dac_out2_pa5>;
```

Reply:
Thanks, good spot. Fixed.